components/admin/application-table.tsx

```diff
@@ -24,7 +24,7 @@ export default function ApplicationTable({
 }: TableProps): JSX.Element {
   return (
     <>
-      {applications.results.length > 0 ? (
+      {applications?.results.length > 0 ? (
```

Comment:
Typescript should probably have noticed this could be null/undefined. If it didn't then it suggests the type of `applications` is wrong somewhere. 
In `TableProps`consider making `applications` optional with a `?`
Also in the function params consider using `applications = []` then you could avoid the `?` here which is a bit neater - it's nicer if you know that values are going to exist.

Reply:
Declared that it could be null and added extra check.